analytics_data_api/constants/educ_level.py

```diff
@@ -0,0 +1,16 @@
+UNKNOWN = 'unknown'
+
+LEVEL_OF_EDUCATION_LOOKUP = {
```

Comment:
@bradenmacdonald Maybe you could rename this to something like `EDUCATION_LEVELS`? There is another constant that is used in the same way and doesn't have a `_LOOKUP` suffix (`CLEANED_GENDERS`). I think it would be good to make the naming consistent.